sheets/3/exercise-3.tex

```diff
@@ -21,14 +21,14 @@
 			\item[] …drehe dich nach links.
 		\end{itemize}
 	\item
-		Laufe das Muster aus b) mithilfe einer for-Schleife 3 Mal.
+		Laufe das Muster aus b) mithilfe einer For-Schleife 3 Mal.
```

Comment:
eine andere idee wäre das for klein zu schreiben aber in ein lstinline zu packen. das es als keyword angezeigt wird.

Reply:
In Aufgabe 1, als die Schleife eingeführt wurde, wurde am häufigsten "For-Schleife" verwendet. Deshalb hab ich das auf dem Blatt in allen Aufgaben zu "For-Schleife" geändert, damit es einheitlich ist.